### PR TITLE
[Feature-#3828][server] Delete recipients and CCs in the process defi…

### LINKED ISSUE #3828: [Feature][server] Delete recipients and CCs in the process definition and select the associated alert group instead (back end)

### PR DESCRIPTION
**Pull request by CalvinKirs**
Delete recipients and CCs in the process definition and select the associated alert group instead

* The specific alarm instance associated with the alert group
* The process definition only associates the alert group, and does not care about the specific alarm instance.
* add sql upgrade script
* delete alert column(show_type、alert_type、re、cc)
* delete alertGroup column(alert_type)
* delete t_ds_process_definition column(receivers、cc) and add column group_warning_id
* delete t_ds_process_definition_version column(receivers、cc) and add column group_warning_id
* delete  t_ds_alert columns( show_type,alert_type, receivers, receivers_cc)

this closes #3828

### CODE REVIEW

**Comment by CalvinKirs** [comment]
No need to pay attention to code style. Here we don’t need to set default

**Comment by CalvinKirs** [comment]
todo The alert group needs to be associated with the alert instance id (delete alert_type)

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/AlertDao.java [hunk 1]
```diff
@@ -99,31 +97,31 @@ public int updateAlert(AlertStatus alertStatus, String log, int id) {
     /**
      * query user list by alert group id
      *
-     * @param alerGroupId alerGroupId
+     * @param alertGroupId alertGroupId
      * @return user list
      */
-    public List<User> queryUserByAlertGroupId(int alerGroupId) {
+    public List<User> queryUserByAlertGroupId(int alertGroupId) {
```
Comment (gaojun2048) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/AlertDao.java:103]: I think alert group and user no longer have any relationship in this feature. So this method need be delete.

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/AlertDao.java [hunk 2]
```diff
@@ -207,11 +193,11 @@ public void sendTaskTimeoutAlert(int alertgroupId, String receivers, String rece
...
      * list user information by alert group id
      *
-     * @param alertgroupId alertgroupId
+     * @param alertGroupId alertGroupId
      * @return user list
      */
-    public List<User> listUserByAlertgroupId(int alertgroupId) {
-        return userAlertGroupMapper.listUserByAlertgroupId(alertgroupId);
+    public List<User> listUserByAlertGroupId(int alertGroupId) {
+        return userAlertGroupMapper.listUserByAlertGroupId(alertGroupId);
```
Comment (gaojun2048) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/AlertDao.java:200]: UserAlertGroupMapper need be delete in this feature.

File: dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/UserAlertGroupMapper.java [hunk 3]
```diff
@@ -30,16 +30,18 @@
...
     /**
-     *  list user by alertgroupId
-     * @param alertgroupId alertgroupId
+     * list user by alertgroupId
+     *
+     * @param alertGroupId alertgroupId
      * @return user list
      */
-    List<User> listUserByAlertgroupId(@Param("alertgroupId") int alertgroupId);
+    List<User> listUserByAlertGroupId(@Param("alertgroupId") int alertGroupId);
```
Comment (gaojun2048) [inline dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/UserAlertGroupMapper.java:45]: UserAlertGroupMapper is need be delete in this feature.

**Review by gaojun2048 — CHANGES_REQUESTED** [review]
1、receivers and receiversCc still in some method`s input params.
2、UserAlertGroupMapper need be delete in this feature.

**Review by gaojun2048 — APPROVED** [review]

**Pull request merged**